shogun-lib/src/main/java/de/terrestris/shogun/lib/service/FileService.java

```diff
@@ -51,6 +65,63 @@ public File create(MultipartFile uploadFile) throws Exception {
         return savedFile;
     }
 
+    /**
+     * Creates a new File entity in the Database.
+     * The file is stored as file on the .
+     *
+     * @param uploadFile
+     * @param writeToSystem
+     * @return
+     * @throws Exception
+     */
+    public File create(MultipartFile uploadFile, Boolean writeToSystem) throws Exception {
+        if (!writeToSystem) {
+            return this.create(uploadFile);
+        }
+
+        String uploadBasePath = uploadProperties.getPath();
+        String filename = uploadFile.getOriginalFilename();
```

Comment:
- I think we should validate that `uploadBasePath` and `filename` aren't empty here (unless that isn't done already at another time)
- do we need to check / sanitize `fileName` if we use the filename as part of the final file path? thinking of e.g. special characters 
    - maybe it's easier and safer to not use the filename at all - if we don't really need it

Reply:
I think most characters are allowed in linux-based systems. but what happens if you modify the request and change `fileName` to something that contains slashes?

- will that create a subfolder?
- will the file creation fail?
- can you somehow break out of the current directory?